tests/conftest.py

```diff
@@ -20,3 +23,19 @@ def event_loop():
 def execute_before_any_test():
     config.SETTINGS.load_and_exit()
     config.SETTINGS.active.server_address = "http://mock"
+
+
+@pytest.fixture(scope="session", autouse=True)
+def clean_env_vars():
+    """Cleans the environment variables before any test is run."""
+    original_values = {}
+    for name in ENV_VARS_TO_CLEAN:
+        original_values[name] = os.environ.get(name)
+        if original_values[name] is not None:
+            del os.environ[name]
+
+    yield
+
+    for name in ENV_VARS_TO_CLEAN:
+        if original_values[name] is not None:
+            os.environ[name] = original_values[name]
```

Comment:
Given that this will run until the session has completed I don't know if there's a point of resetting the environment variables after everything has completed. There should be nothing left that would use them at that point.